src/app/static/jest.config.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
         "json",
         "vue"
     ],
+    "moduleNameMapper": {"tokenize2": "<rootDir>/node_modules/tokenize2/tokenize2.js"},
```

Comment:
The tokenize2 package has its entrypoint specified in error as `/dist/tokenize2.js` (it should be `dist/tokenize2.js`) - webpack resolves this correctly as relative to the package's own directory root but jest treats it as a proper absolute path, hence this additional wiring up in jest.